cmd/manager/api/api.go

```diff
@@ -0,0 +1,28 @@
+package api
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/gorilla/mux"
```

Comment:
That was the plan, just wanted to put some code up in order to get to quick feedback.